cloudinit/ssh_util.py

```diff
@@ -267,6 +269,21 @@ def extract_authorized_keys(username, sshd_cfg_file=DEF_SSHD_CFG):
                         "config from %r, using 'AuthorizedKeysFile' file "
                         "%r instead", DEF_SSHD_CFG, auth_key_fns[0])
 
+        # check if one of the keys is the user's one
+        for key_path, auth_key_fn in zip(key_paths.split(), auth_key_fns):
+            if any([
+                '%u' in key_path,
+                '%h' in key_path,
+                auth_key_fn.startswith(pw_ent.pw_dir)
+            ]):
+                user_authorizedkeys_file = auth_key_fn
+
+        if user_authorizedkeys_file != default_authorizedkeys_file:
+            LOG.debug("AuthorizedKeysFile has an user-specific"
+                                " authorized keys, using " +
+                                user_authorizedkeys_file)
```

Comment:
```suggestion
            LOG.debug(
                "AuthorizedKeysFile has an user-specific"
                " authorized keys, using %s",
                user_authorizedkeys_file
            )
```
Should fix the flake8/pylint issue.

Reply:
Makes sense, also renamed "authorized keys" in "authorized_keys"